themes/bootstrap3/templates/search/bulk-action-buttons.phtml

```diff
@@ -1,11 +1,16 @@
-<?php if (isset($this->showCheckboxes) && $this->showCheckboxes): ?>
+<?php
+  $showBulkOptions = $this->searchSettings($this->params)->bulkOptionsEnabled();
```

Comment:
Would it be cleaner to just set `$searchSettings = $this->searchSettings($this->params);` here and then call the appropriate methods in the checks below? Not sure that there's a lot of value in assigning to variables if they're only used in one place each.

Reply:
Yes, done!